src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
```

Comment:
missing header comments.
how will the user know what does Block mean? how will other developers know how to use this class?